### PR TITLE
Vsphere update + change default k8s to 1.22 from 1.23

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -434,10 +419,10 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1000
       rancher-vsphere-cpi:
-        repo: rancher-charts
+        repo: rancher-rke2-charts
         version: 1.1.000
       rancher-vsphere-csi:
-        repo: rancher-charts
+        repo: rancher-rke2-charts
```
Comment (deniseschannon) [inline channels-rke2.yaml:425]: @phillipsj @rosskirkpat Are these the valid CPI/CSI versions for 1.22 from the rke2-charts?

Comment (phillipsj) [reply channels-rke2.yaml:425]: @deniseschannon yes these look correct. They match the versions that they shipped with that release. https://github.com/rancher/rke2/blob/v1.22.7%2Brke2r2/Dockerfile#L109-L110

File: data/data.json [hunk 2]
```diff
@@ -16541,12 +16297,12 @@
       "version": "0.1.1000"
      },
      "rancher-vsphere-cpi": {
-      "repo": "rancher-charts",
-      "version": "1.1.000"
+      "repo": "rancher-rke2-charts",
+      "version": "1.1.002"
```
Comment (phillipsj) [inline data/data.json:16301]: @deniseschannon these are correct too.

**Review by brandond — DISMISSED** [review]

**Review by phillipsj — DISMISSED** [review]
LGTM!

**Comment by brandond** [comment]
LGTM once the CI chart version checks pass.

**Review by brandond — APPROVED** [review]

**Review by rancher-max — APPROVED** [review]
LGTM

**Comment by deniseschannon** [comment]
https://github.com/rancher/rancher/issues/36987

**Pull request merged**